causal_testing/json_front/README.md

```diff
@@ -0,0 +1,32 @@
+# JSON Causal Testing Framework Frontend
+
+The JSON frontend allows Causal Tests and parameters to be specified in JSON to allow for tests to be quickly written
+whilst retaining the flexibility of the Causal Testing Framework (CTF). 
+
+An example is provided in `examples/poisson` which will be walked through in this README to better understand
+the framework
+
+`examples/poisson/causal_test_setup.py` contains python code written by the user to implement scenario specific features
+such as:
+1. Custom Estimators
+2. Causal Variable specification
+3. Causal test case outcomes
+4. Meta constraint functions
+5. Mapping JSON distributions, effects, and estimators to python objects
+
+`examples/poisson/causal_tests.json` is the JSON file that allows for the easy specification of multiple causal tests.
```

Comment:
We'll need quite a bit more instruction for anyone not already familiar with the package to use this. I suggest making an issue to keep track of this. It could be that a jupyter notebook is is a good next step.